### ISSUE TITLE
Unable to view helm deploment details for Gitlab

### ISSUE THREAD

**Issue by sjdrc** [kontena/lens#4110]
**Describe the bug**
Similar to #2922

When I click the release I expect to see the details of the release.
Instead the spinner just goes indefinitely.

The following error appears in console:
```
instrument.js:109 [JSON-API] request GET http://127.0.0.1:40701/api/v2/releases/default/gitlab {reqInit: {…}, error: JsonApiErrorParsed}
release-details.tsx:102 Uncaught (in promise) JsonApiErrorParsed {error: "", messages: Array(1), isUsedForNotification: false}
loadDetails @ release-details.tsx:102
processTicksAndRejections @ internal/process/task_queues.js:93
async function (async)
loadDetails @ release-details.tsx:101
(anonymous) @ release-details.tsx:70
executeAction @ mobx.esm.js:1128
Reaction @ mobx.esm.js:1112
reactionRunner @ mobx.esm.js:2679
(anonymous) @ mobx.esm.js:2660
runReaction_ @ mobx.esm.js:2289
runReactionsHelper @ mobx.esm.js:2455
reactionScheduler @ mobx.esm.js:2430
(anonymous) @ mobx.esm.js:2468
Wj @ react-dom.production.min.js:244
reactionScheduler @ mobx.esm.js:2467
(anonymous) @ mobx.esm.js:2468
Gh @ /home/sebastien/.config/Lens/extensions/lenscloud-lens-extension/dist/renderer.js:347
reactionScheduler @ mobx.esm.js:2467
runReactions @ mobx.esm.js:2436
endBatch @ mobx.esm.js:2076
reportChanged @ mobx.esm.js:470
set @ mobxreact.esm.js:367
pi @ react-dom.production.min.js:185
ck @ react-dom.production.min.js:269
bk @ react-dom.production.min.js:250
ak @ react-dom.production.min.js:250
Tj @ react-dom.production.min.js:250
Lj @ react-dom.production.min.js:243
(anonymous) @ react-dom.production.min.js:123
exports.unstable_runWithPriority @ scheduler.production.min.js:18
gg @ react-dom.production.min.js:122
jg @ react-dom.production.min.js:123
ig @ react-dom.production.min.js:122
Hb @ react-dom.production.min.js:292
gd @ react-dom.production.min.js:73
sentryWrapped @ helpers.js:72
Show 5 more frames
```

**To Reproduce**
1. Deploy gitlab chart
2. Navigate to Apps > Releases
3. Click on the gitlab release
4. Spinner turns indefinitely

**Expected behavior**
I see the details of the release

**Screenshots**
![image](https://user-images.githubusercontent.com/6028151/138371570-a1828f89-14e2-4aaa-b47e-cf6e31bf97f0.png)

**Environment (please complete the following information):**
- Lens Version: 5.2.5
- OS: Arch Linux
- Installation method (e.g. snap or AppImage in Linux): AppImage

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
See description ebove

**Kubeconfig:**
N/A

**Additional context**
N/A

**Comment by Nokel81** [comment]
Fixed by https://github.com/lensapp/lens/pull/3905, will be part of 5.3.0

**Comment by sjdrc** [comment]
@Nokel81 I've tried out 5.3.0-alpha.6 and that does fix the infinite spinner. However, an error message is now displayed:
```
 Failed to get release details: RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
```

**Comment by Nokel81** [comment]
Ah thanks, that is caused because I guess the helm deployment is larger than 200Kb, would you mind verifying that to be true?

Maybe we could detect this error and ask the user if he wants to try again with a larger buffer (don't want to do it automatically because it might be Gb in size)

**Comment by sjdrc** [comment]
The manifest is 1.4M plaintext 1.9M base64.

**Comment by Nokel81** [comment]
Thanks. I think we can safely increase the limit of that request to several MiB

**Comment by sjdrc** [comment]
Thank you! Will this fix make it into 5.3.0?

**Comment by Nokel81** [comment]
Yes
